timemachine/md/smc.py

```diff
@@ -151,6 +153,12 @@ def effective_sample_size(log_weights):
     return 1 / np.sum(normed_weights ** 2)
 
 
+def jeffective_sample_size(log_weights):
```

Comment:
Probably, the only thing is I can't do the `assert np.isclose(np.sum(normed_weights), 1.0)` in a jax function.

Reply:
Can remove the assertion in this case I think -- but maybe this is something to revisit more generally and carefully next deboggling week...

----

(May want to remove unimportant assertions in functions we would like to `jit` compile, and/or find an idiom that handles the cases when an assertion is important AND we would like to `jit` compile the code that makes the assertion. Perhaps something along the lines of refactoring
```python
def f(x):
   ...
   assert something
   assert something_else
   return result
...
# res = jit(f)(x)  # error
res = f(x)
```
to 
```python
def f(x):
   ...
   all_assertions_passed = something and something_else
   return result, all_assertions_passed
...
result, all_assertions_passed = jit(f)(x)
assert all_assertions_passed
```
, so that the assertion conditions are still computed, but the assertion is moved outside of the `jit` compiled function...)